benchmarks/benchmark.py

```diff
@@ -0,0 +1,68 @@
+'''
+numpy module used for vectorization
+scipy for distance calculation
+ribs for generating centroids to be benchmarked
+'''
+import numpy as np
+from scipy.spatial import distance
+
+from ribs.archives import CVTArchive
+
+
+def get_score(centroids, num_samples, seed):
+    '''
```

Comment:
Make this be a Google-style docstring. See here for more info: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html